lib/src/main/java/com/cloudbees/groovy/cps/impl/CallEnv.java

```diff
@@ -38,6 +39,8 @@
 
     private Invoker invoker;
 
+    int depth;
```

Comment:
After deserialization of a program running prior to the upgrade, this will be 0. Is that expected and safe?

Reply:
Yes -- worst case, calls can run deeper than they're supposed to, but since the limit we set is arbitrary that's not a huge problem.  We just want to cover the unbounded cases.